docs/content/en/getting_started/upgrading.md

```diff
@@ -145,8 +164,6 @@ There is another breaking change regarding the import of SSLyze scans. The parse
 
 Release 2.7.0 contains a beta functionality to make permissions for the configuration of DefectDojo more flexible. When the settings parameter `FEATURE_CONFIGURATION_AUTHORIZATION` is set to `True`, many configuration dialogues and API endpoints can be enabled for users or groups of users, regardless of their **Superuser** or **Staff** status, see [Configuration Permissions]({{< ref "../usage/permissions/#configuration-permissions" >}}).
 
-The functionality using the flag `AUTHORIZATION_STAFF_OVERRIDE` has been removed. The same result can be achieved with giving the staff users a global Owner role. 
```

Comment:
@dsever why did you removed this line from a old release?

Reply:
That is interesting, gitlab was showing to me that those lines are removed in current dev version, I agree it should be returned back